dotCMS/src/main/java/com/dotcms/contenttype/model/field/Field.java

```diff
@@ -244,4 +241,33 @@ public FieldValueRenderer valueRenderer() {
   public FieldValueRenderer listRenderer() {
     return new FieldValueRenderer() {};
   }
+
+  /**
+   * Field' properties to be set by the UI
+   * @return
+   */
+  public Collection<ContentTypeFieldProperties> getFieldContentTypeProperties(){
+    return null;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)